src/plugins/vis_type_vega/public/vega_view/vega_base_view.js

```diff
@@ -230,14 +231,12 @@ export class VegaBaseView {
   }
 
   _addMessage(type, text) {
-    if (!this._$messages) {
-      this._$messages = $(`<ul class="vgaVis__messages">`).appendTo(this._$parentEl);
```

Comment:
"this._$messages" won't be used anymore. Should we remove this private property?